task/max2max/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	_ "github.com/aliyun/aliyun-odps-go-sdk/sqldriver"
+	"github.com/goto/maxcompute-transformation/internal/client"
+	"github.com/goto/maxcompute-transformation/internal/config"
+	"github.com/goto/maxcompute-transformation/internal/loader"
+	"github.com/goto/maxcompute-transformation/internal/logger"
+)
+
+// TODO:
+// - graceful shutdown
+// - error handling
+// - instrumentation
```

Comment:
Can we please add these things.
Also we need to log proper errors in the execution to help with debugging of jobs later.

Reply:
will address this on another PRs